sdk/websites/Microsoft.Azure.Management.WebSites/src/Microsoft.Azure.Management.Websites.csproj

```diff
@@ -7,9 +7,9 @@
 		<PackageId>Microsoft.Azure.Management.Websites</PackageId>
 		<Description>Provides website management capabilities for Microsoft Azure.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Websites</AssemblyName>
-		<Version>3.1.2</Version>
+		<Version>3.2.0</Version>
```

Comment:
Please increase the version to '4.0.0' as breaking changes came into this version

Reply:
Increased the version to `4.0.0`